apis/dataprotection/v1alpha1/backuppolicy_types.go

```diff
@@ -98,15 +98,21 @@ type CommonBackupPolicy struct {
 
 	// which backup tool to perform database backup, only support one tool.
 	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength=63
 	// +kubebuilder:validation:Pattern:=`^[a-z0-9]([a-z0-9\.\-]*[a-z0-9])?$`
 	BackupToolName string `json:"backupToolName,omitempty"`
 }
 
 type PersistentVolumeClaim struct {
 	// the name of the PersistentVolumeClaim.
+	// +kubebuilder:validation:Required
```

Comment:
The `Name` field cannot be set to `Required`, and when Name is not set, it will be automatically retrieved from the environment variable.